aspnetcore/fundamentals/servers/kestrel/diagnostics.md

```diff
@@ -113,3 +113,7 @@ using var badRequestListener = new BadRequestEventListener(diagnosticSource, (ba
 app.MapGet("/", () => "Hello world");
 app.Run();
 ```
+
+## Behavior with debugger attached
+
+Certain timeouts and rate limits aren't enforced when a debugger is attached to a Kestrel process. For more information, see [Behavior with debugger attached](../kestrel.md#behavior-with-debugger-attached).
```

Comment:
This is probably OK but I prefer absolute paths so you can copy/paste that anywhere.